headertest/dummy_header.go

```diff
@@ -31,6 +28,24 @@ type DummyHeader struct {
 	hash header.Hash
 }
 
+func RandDummyHeader(t *testing.T) *DummyHeader {
+	t.Helper()
+
+	dh := &DummyHeader{
+		Raw{
+			PreviousHash: RandBytes(32),
+			Height:       randInt63(),
+			Time:         time.Now().UTC(),
+		},
+		nil,
+	}
+	err := dh.rehash()
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
require?

Reply:
Hmm, not necessary. 
3 > 1 import

Require is not import by headertest